addons/stock/stock.py

```diff
@@ -4596,6 +4602,17 @@ def show_details(self, cr, uid, ids, context=None):
              'context': context,
         }
 
+    def _copy_remaining_pack_lot_ids(self, cr, uid, ids, new_operation_id, context=None):
+        stock_pack_operation_lot_obj = self.pool["stock.pack.operation.lot"]
+
+        for old_operation in self.browse(cr, uid, ids, context=context):
+            for pack_lot_id in old_operation.pack_lot_ids:
+                new_qty_todo = pack_lot_id.qty_todo - pack_lot_id.qty
+
+                if new_qty_todo > 0:
+                    stock_pack_operation_lot_obj.copy(cr, uid, pack_lot_id.id, {'operation_id': new_operation_id,
```

Comment:
A write on the existing would be another possibility (I did something similar in mrp)